ks_patches/create_notification_for_sms.js

```diff
@@ -3,22 +3,12 @@ export function createNotificationForSms(smsMsg, phoneNumber, dateUtils, current
     var enrollment = selections.selectedEnrollment && selections.selectedEnrollment.enrollment;
     if (enrollment) {
         var selectedEnrollment = selections.selectedEnrollment;
-        var today = dateUtils.getToday();
-        var userProfile = sessionStorageService.get('USER_PROFILE');
-        var storedBy = userProfile && userProfile.userCredentials && userProfile.userCredentials.username ? userProfile.userCredentials.username : '';
 
         var noteText = `SMS sendt til ${phoneNumber}: ${smsMsg}`;
         var newNote = {
             value: noteText,
-            storedDate: dateUtils.formatFromUserToApi(today),
-            displayDate: today,
-            storedBy: storedBy
         };
-        if (selectedEnrollment.notes) {
-            selectedEnrollment.notes.splice(0, 0, newNote);
-        } else {
-            selectedEnrollment.notes = [newNote];
-        }
+        selectedEnrollment.notes = [newNote];
```

Comment:
Would this not always replace all the notes? So you would always replace the notes list with a one-item list each time a note is added?